en/docs/install-and-setup/setup/setting-up-databases/changing-default-databases/changing-to-postgresql.md

```diff
@@ -195,3 +195,21 @@ Follow the instructions below to change the type of the default datasource.
 
     !!! note
         To give the Key Manager, Publisher, and Developer Portal components access to the user management data with shared permissions, JDBCUserStoreManager has been configured by default. For more information, refer [Configuring Userstores]({{base_path}}/administer/product-administration/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-jdbc-user-store).
+
+    !!! info
+        **Changing WSO2CARBON_DB to PostgreSQL**
+
+        By default `WSO2CARBON_DB` will be an embedded H2 database and it is **not necessary** to change it to another database. But if you have a requirement to change it, you can follow the below steps. (When changing the carbon database, make sure that **each server node have its own WSO2CARBON_DB**. If you don't want to change the carbon database, then you can ignore this section.)
+
+        - Create tables in the carbon database (`WSO2CARBON_DB`) using the script `<API-M_HOME>/dbscripts/postgresql.sql`.
+        -   Open the `<API-M_HOME>/repository/conf/deployment.toml` configuration file. Locate the `[database.local]` configuration element and update the URL pointing to your PostgreSQL database, the username, and password required to access the database and the PostgreSQL driver details similarly as explained before.
```

Comment:
is there a formatting issue here? (with - )

Reply:
No
![Screenshot from 2020-12-21 14-06-06](https://user-images.githubusercontent.com/8374895/102756368-ca690380-4395-11eb-9faf-7a8e3924f2ee.png)